src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -51,6 +51,16 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "TOFBarrelRecHit",
             "TOFEndcapRecHits",
 
+            // DRICH
+            "DRICHRawHits",
+            "DRICHRawHitsAssociations",
+            "DRICHAerogelTracks",
+            "DRICHGasTracks",
+            "DRICHMergedTracks",
+            "DRICHAerogelIrtCherenkovParticleID",
+            "DRICHGasIrtCherenkovParticleID",
+            // "DRICHMergedIrtCherenkovParticleID",
```

Comment:
Let's drop the redundant (merged) ones.
```suggestion
            "DRICHAerogelIrtCherenkovParticleID",
            "DRICHGasIrtCherenkovParticleID",
```